src/consts/tokens.ts

```diff
@@ -38,12 +38,13 @@ export const tokenList: WarpTokenConfig = [
 
   // Neutron token
   {
-    type: 'native',
+    type: 'collateral',
```

Comment:
Does this work? The CwHypCollateral adapter in Yorke's branch is not yet implemented.